src/Components/FormInput/formInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './formInput.module.css'
+
+export const FormInput = ({ labelText, type, value, onChange }) => {
+  return (
+    <div className={styles.formDiv}>
+      <label className={styles.formLabel}>{labelText}</label>
```

Comment:
It's better to wrap `input` in `label` to link it or use `for` attribute to link label and input together